### PR TITLE
Added guidelines for commit tagging

### PR DESCRIPTION
**Pull request by wagenet**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -162,3 +161,33 @@ Code words are:
...
+
+# Appendix
+
+## Commit Tagging
+
+All commits should be tagged. Tags are denoted by square brackets (`[]`) and come at the start of the commit message.
+
+### Bug Fixes
+
+In general bug fixes are pulled into the beta branch. As such, the prefix is: `[BUG beta]`. If a bug fix is a serious regression that requires a new patch relese, `[BUG release]` can be used instead.
```
Comment (rwjblue) [inline CONTRIBUTING.md]: The prefix is "BUGFIX", so it would be: `[BUGFIX canary]`

**Comment by rwjblue** [comment]
Other than that small correction, LGTM.

File: CONTRIBUTING.md (hunk 1 above)
Comment (wagenet) [inline CONTRIBUTING.md]: Whoops, fixed.

**Pull request merged**